jujugui/static/gui/src/app/init/topology/service.js

```diff
@@ -1546,11 +1546,6 @@ class ServiceModule {
    * @method createServiceNode
    */
   createServiceNode(node, self) {
-    var staticURL = self.topo.staticURL || '';
```

Comment:
on line 1786 there are some paths that reference a static path, should those be updated to use this staticURL?

Reply:
Thanks!